### ISSUE TITLE
1/2 mapping not reflected in the NDX file

### ISSUE THREAD

**Issue by ricalessandri** [marrink-lab/pycgbuilder#5]
The atoms which are (very nicely!) displayed as belonging to two beads at the same time, should also have a weight of 1/2 in the resulting NDX file. Instead, this does not seam to be the case. For example, the mapping of toluene (Cc1ccccc1) should be:

```
[ B0 ]
1 8 10 9 2 1 8 10 9 2    3 7

[ B1 ]
4 12 4 12    3 5

[ B2 ]
14 6 14 6    5 7
```
where the atoms number **3**, **5**, and **7** are the ones which are shared, and appear only once instead of twice in each bead (i.e., their weight is 1/2).

Instead, although the `pycgbuilder` GUI shows a nicely shared atom, the resulting NDX file is:
```
[ BD0 ]
1 2 3 7 8 9 10

[ BD1 ]
3 4 5 12

[ BD2 ]
5 6 7 14
```
where **3**, **5**, and **7** have the same weight in each group as the other atoms.

**Comment by pckroon** [comment]
Since NDX doesn't really support weights (it has to be hacked around as shown) this will very quickly spiral out of control I'm afraid. What if you have a bead in which atom A is 1/2, atom B is 1/3, atom C is 1/5, and atom D is 1? I think D would have to be listed 30 times! Granted, 1/5 atom weights are not to be expected, but the point stands. 
As such, I think this is a won't fix. Does it cause large problems? You get coordinates with proper weighing in the pdb file, and the map file contains the correct weights.
